pkg/quota/evaluator/core/pods.go

```diff
@@ -337,6 +366,23 @@ func isTerminating(pod *api.Pod) bool {
 	return false
 }
 
+func doesPriorityClassExist(pod *api.Pod) bool {
+	return pod.Spec.PriorityClassName != ""
+}
+
+func isPriorityClassIn(pod *api.Pod, scope string) bool {
+	if pod.Spec.PriorityClassName == "" {
+		return false
+	}
+	classes := strings.Split((strings.Split(scope, ":")[1]), ",")
```

Comment:
i am sorry for missing this earlier.

i am concerned that the `,` delimiter won't work when using the CLI.

for example:

```
$ kubectl create quota myquota --hard=pods=10 --scopes=BestEffort,Terminating,PriorityClassNameIn:foo,bar
```

this won't work.  we probably need to stick with a : delimiter.